ArduPlane/ArduPlane.cpp

```diff
@@ -60,7 +60,7 @@ const AP_Scheduler::Task Plane::scheduler_tasks[] = {
     SCHED_TASK(compass_cal_update,     50,    50),
     SCHED_TASK(accel_cal_update,       10,    50),
 #if OPTFLOW == ENABLED
-    SCHED_TASK(update_optical_flow,    50,    50),
+    SCHED_TASK_CLASS(OpticalFlow, &plane.optflow, update,    50,    50),
```

Comment:
Not to scope creep it, but I notice that copter runs optical flow at 200Hz. Given that quadplanes might want to be using optical flow as well should we be raising this to a 200Hz task on plane as well?

Reply:
Fair enough. I don't have the hardware so I can't volunteer...